docs/src/standards/rfcs/rfc9380.md

```
@@ -3034,7 +3034,7 @@ follows:
 
 - Z: -(2 + I)
 
-- 
+-
```

markdownlint-cli2 (0.22.1)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 0

(MD005, list-indent)